samples/admin/ADMINReqMoveComplete/ADMINReqMoveComplete.c

```diff
@@ -0,0 +1,500 @@
+	
+    /* Initialize the notes. */
+    if (error = NotesInitExtended(argc, argv))
+    {
+        PRINTERROR (error,"NotesInitExtended");
+        return(1);
+    }
+	
+    /* Reading Domino data directory. */
+    OSGetDataDirectory(szDataPath);
+    PRINTLOG("data path : %s\n", szDataPath);
+
+    /* Adding trailing path separator. */
+    if (error = OSPathAddTrailingPathSeparator(szDataPath, sizeof(szDataPath)-1))
+    {
+        PRINTERROR(error,"OSPathAddTrailingPathSeparator");
+        NotesTerm();
+        return (1);
+    }
+    PRINTLOG("data path with trailing path separator: %s\n", szDataPath);
+
+    /* Preparing path for cert ID. */
+    strncat(szDataPath, szOrgUnitCertID, sizeof(szDataPath)-1 - strlen(szDataPath));
+    pszCertIDFilePath = szDataPath;
+    PRINTLOG("cert ID path : %s\n", pszCertIDFilePath);
+
+    /* Construct the path for the admin request file. */
+    if (error = OSPathNetConstruct(NULL, pszServerName, pszAdmin4NSFFile, szAdminFilePath))
+	{
+        PRINTERROR(error,"OSPathNetConstruct");
+        NotesTerm();
+        return (1);
+    }
+	
+    /* Open the admin4 DB database. */
+    if (error = NSFDbOpen(szAdminFilePath, &db_handle))
+    {	
+	PRINTERROR (error,"NSFDbOpen");
+	NotesTerm();
+	return (1);
+    }
+	
+     /* Construct the path for the NAB file */
+    if (error = OSPathNetConstruct(NULL, pszServerName, pszNABFile, szNABFilePath))
+    {
+        PRINTERROR(error,"OSPathNetConstruct");
+        NotesTerm();
+        return (1);
+    }
+
+    PRINTLOG("*************************************************\n");
+    PRINTLOG("***********Testing-ADMINReqMoveUserInHier********\n");
+    PRINTLOG("*************************************************\n");
+
+    /* Open the names.nsf (NAB) database. */
+    if (error = NSFDbOpen(szNABFilePath, &hNABook))
+    {
+        PRINTERROR (error,"NSFDbOpen");
+        NotesTerm();
+        return (1);
+    }
+
+    /* Get the Note ID of the user to be renamed. */
+    if (error = REGFindAddressBookEntry (hNABook, szUserFld, szUserName, &noteID))
+    {
+        NSFDbClose (hNABook);
+        PRINTERROR (error,"REGFindAddressBookEntry");
+        NotesTerm();
+        return (1);
+    }
+
+    /* Get the Note handle. */
+    if (error = NSFNoteOpen(hNABook, noteID, 0, &nhNote))
+    {
+        NSFDbClose (hNABook);
+        PRINTERROR (error,"NSFNoteOpen");
+        NotesTerm();
+        return (1);
+    }
+
+    /* Get the current certifier context. */
+    if (error = GetCertCtx(pszCertIDFilePath, &hCertCtx, szPassword))
+    {
+	NSFNoteClose (nhNote);
+	NSFDbClose (db_handle);
+	PRINTERROR (error,"GetCertCtx");
+	NotesTerm();
+	return (1);
+    }
+
+    /* Intializing the ADMINReqParams structure */
+    memset(&ARPptr, 0x00, sizeof(ARPptr));
+
+    if (error = ADMINReqMoveUserInHier ( hCertCtx,
+                               hNABook,
+                               nhNote,
+                               "O=HCLPNP", /* Target Certifier */
```

Comment:
Is certifier be dynamic instead of hardcoded.

Reply:
Let it be hard coded as of now.